package_check.sh

```diff
@@ -248,8 +252,8 @@ elif [ ${product} = "pxc56" -o ${product} = "pxc57" ]; then
     fi
   fi
 
-elif [ ${product} = "pxc80" ]; then
-  echo "Package check for PXC-80 is not implemented!"
+elif [[ ${product} =~ ^pxc8[1-9]{1}$ ]]; then
```

Comment:
Should not we have `^pxc8[0-9]{1}$ ]]` ? Otherwise the test will fail for 80.

Reply:
I think we don't have pxc package check implemented for pxc 80 so I have started it with 81